examples/resources/serverless/functions.yaml

```diff
@@ -1,62 +1,221 @@
+                    {
+                      "source": "$item.name",
+                      "name": "code.env.name",
+                    }, {
+                      "source": "$item.value",
+                      "name": "code.env.value"
+                    }, {
+                      "name": "code.env.valueFrom",
+                      "widget": "ResourceLink",
+                      "source": "$item.valueFrom",
+                      "linkText": "$exists(data.configMapKeyRef)? data.configMapKeyRef.name : data.secretKeyRef.name",
+                      "resource": {
+                        "name": "$exists(data.configMapKeyRef) ? data.configMapKeyRef.name : data.secretKeyRef.name",
+                        "namespace": "root.metadata.namespace",
+                        "kind": "$exists(data.configMapKeyRef)
+                        ? 'ConfigMap' 
+                        : 'Secret'"
+                      }
+                    }, {
+                      "name": "code.env.controlledBy",
+                      "widget": "ControlledBy",
+                      "source": "$item.owners"
+                    }, {
+                      "name": "code.env.source",
+                      "widget": "Badge",
+                      "source": "$exists($item.valueFrom.configMapKeyRef)
+                        ? 'CONFIG MAP' 
+                        : $exists($item.valueFrom.secretKeyRef)
+                          ? 'SECRET'
+                          : 'CUSTOM'"
+                    }, {
+                      "name": "code.env.key",
+                      "source": "$exists($item.valueFrom.configMapKeyRef)
+                        ? $item.valueFrom.configMapKeyRef.key 
+                        : $exists($item.valueFrom.secretKeyRef)
+                          ? $item.valueFrom.secretKeyRef.key 
+                          : 'N/A'"
+                    }
+                  ]
+                }, {
+                  "widget": "ResourceList",
+                  "source": "$replicas()",
+                  "name": "code.replicas",
+                  "disableCreate": true
+                }
+              ]
             },
             {
-              "name": "runtime",
-              "source": "spec.runtime"
-            },
-            {
-              "name": "runtimeImageOverride",
-              "source": "spec.runtimeImageOverride"
-            },
-            {
-              "name": "Configured",
-              "source": "status.conditions[?(@.type=='ConfigurationReady')].status"
-            },
-            {
-              "name": "Built",
-              "source": "status.conditions[?(@.type=='BuildReady')].status"
-            },
+              "name": "tabs.configuration",
+              "children": [ 
+                {
+                  "widget": "ResourceList",
+                  "source": "$apiRules()",
+                  "name": "configuration.apiRules"
+                }, {
+                  "widget": "ResourceList",
```

Comment:
After the creation of the Subscription from the Function details view, the user is redirected to subscription details. Instead, it should stay on the function detail page.